### DISCUSSION TITLE
Can I configure tag dialog to always sign with default GPG?

### DISCUSSION THREAD

**Discussion by nul800sebastiaan** [gitextensions/gitextensions#10151] [Q&A]
I've searched high and low but I think it's currently impossible for the tag dialog to be configure to pick a different dropdown by default. I would really like "Sign with default GPG" to always be selected when I create a new tag. Possible now, or new feature request? 😄 

![image](https://user-images.githubusercontent.com/304656/184541526-569201c4-53e0-4f50-a107-17c794c66ae0.png)

**Comment by gerhardol** [comment]
No that is not possible. GitExtensions already have more settings than most users can manage, so I personally would not prefer a user configuration here.
To remember last used value is a reasonable feature though.
Similar question in #990.

It is not possible to see the GPG info in GE, See also #10141.

**Reply by nul800sebastiaan** [reply]
Thanks @gerhardol! As a OSS maintainer myself, I totally understand, more config options is usually asking for more trouble 😅

I really should have worded it differently, I distinctly remember that I didn't use to get an additional popup for a lightweight tag.. Why do I need to enter another message in a text editor?

https://user-images.githubusercontent.com/304656/184588181-1469d08a-cc2d-499f-aa8d-0d260395c57a.mp4

The friction is just a bit frustrating, all of the other options in the dropdown work without popping up another editor.. and that's why I wanted to use one of those instead.

**Reply by gerhardol** [reply]
The console is from the Git command. Maybe it can be provided as a file to the Git command (like commit message is handled)

**Reply by RussKie** [reply]
This form could use some love and make over, and  I'd welcome PRs to enhance the dialog. 

I don't see a problem with getting the last selection persisted, though I wouldn't expose this value as a configurable setting anywhere.
There's another problem with dialog is that it doesn't allow specifying which remote to push the tag to, unlike the "Delete Tag" dialog.

**Reply by gerhardol** [reply]
https://git-scm.com/docs/git-tag#Documentation/git-tag.txt---sign
Have you set `tag.gpgSign`?
https://git-scm.com/docs/git-config#Documentation/git-config.txt-taggpgSign

With lightweight tags, no git arg is set and your Git settings will be overridden. Here --no-sign should be used or `git -c tag.gpgSign=false` be set.
